integration_tests/helpers/mergeMiningProxyClient.js

```diff
@@ -23,7 +23,7 @@ class MergeMiningProxyClient {
                 "reserve_size": 60
             }
         });
-        console.log(res.data);
+        console.log("Blocktemplate:",res.data.result.blocktemplate_blob);
```

Comment:
Probably this print can be commented out as a default and uncommented if someone needs debug info